src/analysis/retail/shaman/enhancement/modules/talents/ElementalAssault.tsx

```diff
@@ -118,16 +119,23 @@ class ElementalAssault extends Analyzer {
   }
 
   statistic() {
-    const eaRanks = this.selectedCombatant.getTalentRank(TALENTS_SHAMAN.ELEMENTAL_ASSAULT_TALENT);
+    const totalMaelstrom = this.maelstromWeaponGained;
```

Comment:
this variable is unnecessary, just directly reference the member variable

Reply:
Technically it saves one call to to the getter (unless typescript is doing some kind of caching I'm unaware of)

Reply:
Your getters are pretty expensive actually, it might be worth cacheing them with memoization if you're worried about them being called multiple times that way you can avoid the extra store. Probably fine to leave as is tho